docs/components/bottom_navigation.md

```diff
@@ -7,18 +7,30 @@ iconId:
 path: /components/bottom-navigation/
 -->
 
-
 # Bottom navigation
 
 [Bottom navigation](https://material.io/components/bottom-navigation/#) bars allow movement between primary destinations in an app.
 
-![Example bottom navigation bar with four icons along the bottom: favorites, music, places, and news. The music icon is selected](assets/bottom_navigation/bottom-nav-generic.png)
+![Example bottom navigation bar with four icons along the bottom: favorites, music, places, and news. The music icon is selected](assets/bottom_navigation/Bottomnavigation_hero.png)
+
+## Contents
 
+- [Bottom navigation](#bottom-navigation)
+  - [Contents](#contents)
+  - [Using bottom navigation](#using-bottom-navigation)
+  - [Bottom navigation example](#bottom-navigation-example)
+    - [Anatomy and key properties](#anatomy-and-key-properties)
+    - [Container attributes](#container-attributes)
+    - [Inactive icon attributes](#inactive-icon-attributes)
+    - [Inactive text label attributes](#inactive-text-label-attributes)
+    - [Active icon attributes](#active-icon-attributes)
+    - [Active text label attributes](#active-text-label-attributes)
```

Comment:
We're not including this level of information in the TOC

Reply:
Done